vaadin-simple-grid-filter/src/main/java/software/xdev/vaadin/FilterComponent.java

```diff
@@ -938,10 +931,21 @@ private void removeChipBadgeCondition(final ChipBadgeExtension<FilterCondition<T
 	{
 		this.chipBadges.remove(chip);
 		this.hlChipBadges.remove(chip);
+		// Have to set all filter when removing a condition again because we cannot remove just one filter from the
+		// grid
 		this.updateGridFilter();
 		this.removeQueryParameter(chip);
 	}
 	
+	private void updateGridFilter()
```

Comment:
Method is called ``updateGridFilter`` but it just removes filters and throws a ``FilterComponentResetGridFilterEvent``.

Should likely be called ``resetGridFilters`` or?